### PR TITLE
feat: Update eslint-config-cozy-app

### PR DESCRIPTION
**Pull request by acezard**
We were a bit behind in versions, bugs started to leak in our lints.
These version upgrades should fix them,
but they also have a small chance to detect new lint errors.

### CODE REVIEW

File: packages/eslint-config-cozy-app/package.json [hunk 1]
```diff
@@ -19,17 +19,16 @@
...
-    "@babel/eslint-parser": "^7.17.0",
-    "@typescript-eslint/eslint-plugin": "^5.19.0",
-    "@typescript-eslint/parser": "^5.19.0",
-    "eslint": "^8.13.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.5.0",
-    "eslint-plugin-vue": "^8.6.0",
```
Comment (trollepierre) [inline packages/eslint-config-cozy-app/package.json:31]: isn't it a breaking change to remove plugin vue ?

Comment (acezard) [reply packages/eslint-config-cozy-app/package.json:31]: @trollepierre I guess it is, but do we even have a Vue app? It seems legacy

Comment (acezard) [reply packages/eslint-config-cozy-app/package.json:31]: @trollepierre added breaking change in commit description

**Review by trollepierre — APPROVED** [review]

File: packages/eslint-config-cozy-app/package.json (hunk 1 above)
Comment (trollepierre) [reply packages/eslint-config-cozy-app/package.json:31]: I wonder if we should ask to front team about their opinion. 
With a breaking change, there is no way we are faulty

**Pull request merged**